bigquery_etl/backfill/validate.py

```diff
@@ -6,20 +6,13 @@
 from ..backfill.parse import DEFAULT_REASON, DEFAULT_WATCHER, Backfill, BackfillStatus
 
 
-def validate_duplicate_entry_dates(entry_1: Backfill, entry_2: Backfill) -> None:
+def validate_duplicate_entry_dates(backfill_entry: Backfill, backfills: list) -> None:
     """Check if backfill entries have the same entry dates."""
-    if entry_1.entry_date == entry_2.entry_date:
-        raise ValueError(f"Duplicate backfill with entry date: {entry_1.entry_date}.")
-
-
-def validate_overlap_dates(entry_1: Backfill, entry_2: Backfill) -> None:
-    """Check overlap dates between two backfill entries."""
-    if max(entry_1.start_date, entry_2.start_date) <= min(
-        entry_1.end_date, entry_2.end_date
-    ):
-        raise ValueError(
-            f"Existing backfill entry with overlap dates from: {entry_2.entry_date}."
-        )
+    for backfill_entry_1 in backfills:
+        if backfill_entry.entry_date == backfill_entry_1.entry_date:
```

Comment:
nit: maybe something like `if backfill_entry.entry_date in (b.entry_date for b in backfills)`?